app/assets/javascripts/components/dashboard/navigation/panels/signals/welcome_panel.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
```

Comment:
In ES6 you can export a `default`. When importing the default, you don't need wrap it in curly brackets:

```js
import React from 'react';
```

When importing non-defaults, you have wrap it in curly brackets and you can import as many things as you'd like:

```js
import { Component, PropTypes } from 'react`
```

You can even combine defaults with non-defaults on the same line:

```js
import React, { Component, PropTypes } from 'react`;
```